HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb

```diff
@@ -0,0 +1,13 @@
+hours = gets.to_i
+minutes = gets.to_i
+# this method turns the time into an angle between the watch strips
+def calc_angel(hours, minutes)
+  angel = (minutes * 6 - (hours * 60 + minutes) * 0.5).abs
```

Comment:
Why don't you want to use something like this?
```suggestion
  angel = (minutes * 6 - (hours * 30 + minutes * 0.5)).abs
```

Reply:
There is a difference?